diff --git a/build.gradle b/build.gradle
index e06dddb..6cbd6cb 100644
--- a/build.gradle
+++ b/build.gradle
@@ -3,12 +3,20 @@ apply plugin: 'application'
 apply plugin: 'eclipse'
 
 sourceCompatibility = 11
-version = '0.1'
+version = '1.0'
+configurations.implementation.setCanBeResolved(true)
 jar {
     manifest {
-        attributes 'Implementation-Title': 'WALA Start',
-                   'Implementation-Version': archiveVersion
+        attributes 'Implementation-Title': 'JSONCallGraph',
+                   'Implementation-Version': version,
+                   'Main-Class': 'com.ibm.wala.examples.drivers.JSONCallGraph'
     }
+    
+        doFirst {
+        duplicatesStrategy = 'include'
+        from { configurations.implementation.collect { it.isDirectory() ? it : zipTree(it) } } 
+    }
+    exclude 'META-INF/*.RSA', 'META-INF/*.SF','META-INF/*.DSA' 
 }
 
 repositories {
@@ -30,6 +38,7 @@ dependencies {
     implementation "com.ibm.wala:com.ibm.wala.cast.java.ecj:${walaVersion}"
     implementation "com.ibm.wala:com.ibm.wala.dalvik:${walaVersion}"
     implementation "com.ibm.wala:com.ibm.wala.scandroid:${walaVersion}"
+    implementation "com.googlecode.json-simple:json-simple:1.1.1"
     testImplementation group: 'junit', name: 'junit', version: '4.+'
 }
 
@@ -37,5 +46,5 @@ if (project.hasProperty('mainClass')) {
     mainClassName = project.getProperty('mainClass')
 } else {
     // use a default
-    mainClassName = "com.ibm.wala.examples.drivers.PDFTypeHierarchy"
+    mainClassName = "com.ibm.wala.examples.drivers.JSONCallGraph"
 }
